cli/access-permissions.sh

```diff
@@ -11,11 +11,18 @@ if [ "$(id -u)" -ne 0 ]; then
 	exit 3
 fi
 
+# If specified, only modify the data and extension dirs
+if [ "${1:-}" = "--only-userdirs" ]; then
+	to_update="./data ./extensions"
```

Comment:
It would be nice to include the `DATA_PATH` environment variable as target, if defined:
https://github.com/FreshRSS/FreshRSS/blob/9faf2c1fa3c8c3ed82ff4c95733a018d0f9977a7/constants.php#L63-L69